README.md

```diff
@@ -87,10 +87,16 @@ Modern Paste is intended for system administrators who wish to host their own in
    + `FLASK_SECRET_KEY` - For security reasons, replace the string here with the output of `os.urandom(32)` from a Python shell.
 
 5. **Build the app.**
-   At this point, before continuing, ensure you have the MySQL and Python dev packages installed. This corrects common errors during the `make dependencies` stage of the build.
+   At this point, before continuing, ensure you have the MySQL and Python dev packages installed. This corrects common errors during the `make dependencies` stage of the build. To avoid conflicts with the system Python's modules, it is recommended to use [pyenv](https://github.com/yyuu/pyenv) so the app's Python dependencies remain segregated from the base system's environment. 
```

Comment:
The linter was unhappy with the trailing whitespace on this line. Please just remove the space character after the last period, and everything should be good to go.